packages/@okta/i18n/src/properties/login.properties

```
@@ -399,7 +399,7 @@ enroll.yubikey.subtitle = Insert your YubiKey into a USB port and tap it to gene
 # Enroll TOTP
 enroll.totp.title = Setup {0}
 enroll.totp.selectDevice = Select your device type
-enroll.totp.downloadApp = Download <a href="{0}" class="inline-link">{1} from the {2}</a> onto your mobile device.
+enroll.totp.downloadApp = Download <a href="{0}" target="_blank" rel="noreferer noopener"= class="inline-link">{1} from the {2}</a> onto your mobile device.
```

FYI - this is invalid HTML

```suggestion
enroll.totp.downloadApp = Download <a href="{0}" target="_blank" rel="noreferer noopener" class="inline-link">{1} from the {2}</a> onto your mobile device.
```